codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/route53/Route53InvalidBatchErrorIntegration.kt

```diff
@@ -13,17 +11,15 @@ import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
 import software.amazon.smithy.swift.codegen.integration.SectionWriter
 import software.amazon.smithy.swift.codegen.integration.SectionWriterBinding
 import software.amazon.smithy.swift.codegen.integration.SwiftIntegration
-import software.amazon.smithy.swift.codegen.model.expectShape
 
 class Route53InvalidBatchErrorIntegration : SwiftIntegration {
```

Comment:
Skip this file, it's disabled for now.  Will come back to it.

Reply:
This integration doesn't work.

https://github.com/awslabs/aws-sdk-swift/issues/1027 tracks correcting it.

I will leave it disabled here since it doesn't serve any function.